USING_IMPLEMENTATION_GUIDES.md

```diff
@@ -0,0 +1,151 @@
+# Using FHIR Implementation Guides
+
+An [Implementation Guide (IG)](https://www.hl7.org/fhir/implementationguide.html) is a set of rules that describe how FHIR resources should be used to solve a particular problem. Using IGs, you can make your FHIR server compliant for country-specific set of rules. IGs can also describe a proper way to represent particular content in FHIR (for example, the breast cancer medical diagnostic process).
+
+FHIR is a platform specification. It describes a general set of capabilities that can be used to solve many healthcare data exchange problems.
```

Comment:
Remove this line. On the latest revision of the docs we discarded the general FHIR description and focused only on IGs
```suggestion
```

Reply:
fine. changed